documentation/latest/dev/guide.md

```diff
@@ -70,24 +70,35 @@ kube-system          kube-scheduler-kind-control-plane            1/1     Runnin
 local-path-storage   local-path-provisioner-75f5b54ffd-9kzhm      1/1     Running     0              3h5m
 ```
 
-Finally, retrieve and export the cluster IP address:
+Retrieve and export the cluster IP address:
 
 ```bash
 export SERVER_IP=$(kubectl get nodes kind-control-plane -o json  |  jq -r '[.status.addresses[] | select(.type=="InternalIP")] | .[0].address')
 ```
 
+Finally, use the value of `$SERVER_IP` to configure services hosts in your
+`/etc/hosts` file:
```

Comment:
@gciavarrini I think this is the missing piece that prevented me from working with kind without port forwarding.

Reply:
I didn't have, prior to https://github.com/parodos-dev/parodos/pull/376 , any entry in my `/etc/hosts` 🤷